test/integration/check_navbar_links.js

```diff
@@ -7,10 +7,11 @@
 var tap=require('tap');
 var seleniumWebdriver = require('selenium-webdriver');
 
-//chrome driver
-var driver = new seleniumWebdriver.Builder().withCapabilities(seleniumWebdriver.Capabilities.chrome()).build();
+//Passing in the driver as a variable?
+var driver = process.env.DRIVER || seleniumWebdriver.Builder().withCapabilities(seleniumWebdriver.Capabilities.chrome()).build();
```

Comment:
I don't think this would work... the proces.env docs said that by default passed-in variables are strings, so I need to look into how to do this properly, but I just wanted to illustrate what I was trying to ask about when I mentioned passing in the drivers as variables...